applications/nrf5340_audio/src/bluetooth/bt_management/advertising/bt_mgmt_adv.c

```diff
@@ -154,19 +155,19 @@ static int direct_adv_create(bt_addr_le_t addr)
 	ext_adv_param.options |= BT_LE_ADV_OPT_DIR_ADDR_RPA;
 
 	/* Clear ADV data set before update to direct advertising */
-	ret = bt_le_ext_adv_set_data(ext_adv, NULL, 0, NULL, 0);
+	ret = bt_le_ext_adv_set_data(ext_adv[ext_adv_index], NULL, 0, NULL, 0);
 	if (ret) {
 		LOG_ERR("Failed to clear advertising data. Err: %d", ret);
```

Comment:
Not sure if we show the ext_adv_index when print error log could do help or not. We can discuss.

Reply:
Not until we have multiple advertising sets for CIS at least. It doesn't hurt to include it either, will add it